server/src/main/java/com/coolSchool/CoolSchool/controllers/AuthenticationController.java

```diff
@@ -38,9 +34,15 @@ public ResponseEntity<AuthenticationResponse> authenticate(
 
     @PostMapping("/refresh-token")
     public ResponseEntity<AuthenticationResponse> refreshToken(
-            HttpServletRequest request,
-            HttpServletResponse response
+            @RequestBody RefreshTokenBodyDTO refreshTokenBody
     ) throws IOException {
-        return ResponseEntity.ok(authenticationService.refreshToken(request, response));
+        return ResponseEntity.ok(authenticationService.refreshToken(refreshTokenBody));
+    }
+
+    @PostMapping("/me")
+    public ResponseEntity<AuthenticationResponse> getMe(
+            @RequestBody AccessTokenBodyDTO accessTokenBodyDTO
+    ) {
+        return ResponseEntity.ok(authenticationService.me(accessTokenBodyDTO));
```

Comment:
Don't format like that. See how other controllers are formatted. It's unreadable